ios/FluentUI/Vnext/List/ListHeaderFooter.swift

```diff
@@ -6,26 +6,27 @@
 import UIKit
 import SwiftUI
 
-/// View for List Header (availble for iOS 14.0+)
 struct Header: View {
-    let title: String
-    var tokens: MSFListTokens
+    let state: MSFListSectionState
+    var tokens: MSFListHeaderFooterTokens
 
-    init(title: String, tokens: MSFListTokens) {
-        self.title = title
-        self.tokens = tokens
+    init(state: MSFListSectionState) {
```

Comment:
any specific reason to pass additional data? The header doesn't seem to use anything in addition to title and tokens

Reply:
Eventually I will be passing in the rest of the HeaderFooter data, including footer data, accessory data , # of lines, handler etc! :)